### PR TITLE
Student achievements view

### PR DESCRIPTION
**Pull request by marcinz99**

### CODE REVIEW

File: src/main/java/upeldev/com/github/upel3/controllers/AchievementsController.java [hunk 1]
```diff
@@ -55,9 +57,16 @@ public String showAchievements(Model model,
         model.addAttribute("achievements", achievements);
         model.addAttribute("course", course);
 
-        if(course.getLecturers().contains(currentUser)) return "achievements_lecturer";
-        else return "error";
+        if(course.getLecturers().contains(currentUser) || currentUser.getRoles().contains(Role.ADMIN)){
+            return "achievements_lecturer";
+        }
+        else if(course.getEnrolledStudents().contains(currentUser)){
+            List<StudentAchievement> achieved = studentAchievementService.findAllByUserAndCourse(currentUser, course);
```
Comment (anosek-an) [inline src/main/java/upeldev/com/github/upel3/controllers/AchievementsController.java:64]: Filter achieved list by isAchieved field - now if the achievement isn't satisfied, it still gets displayed.

**Review by anosek-an — CHANGES_REQUESTED** [review]
One small improvement and it'll be all good :D

File: src/main/java/upeldev/com/github/upel3/controllers/AchievementsController.java (hunk 1 above)
Comment (marcinz99) [reply src/main/java/upeldev/com/github/upel3/controllers/AchievementsController.java:64]: Just added `th:if="${achievement.isAchieved()}"`, which kinda does the same thing ultimately

**Review by wyder110 — APPROVED** [review]

**Review by anosek-an — APPROVED** [review]
Good solution, thanks :D

**Pull request merged**